### PR TITLE
Import constants from external crate

### PR DESCRIPTION
**Pull request by n00m4d**
# What
This PR adds import constants from external crate instead of bubblegum program.
# Why
We already import bubblegum client crate so it doesn't make sense to import program crate as well.

### CODE REVIEW

File: src/pubkey_util.rs [hunk 1]
```diff
@@ -1,20 +1,28 @@
...
 use solana_sdk::pubkey::Pubkey;
 
+use crate::errors::BatchMintError;
+
 // todo: import from package with staking/rewards constants
 pub const REWARD_POOL_ADDRESS: Pubkey = pubkey!("J9iTArkeHKahfAiKcFYKK128EC3rBr8ZyVthCE7TE6F9");
 
-pub fn get_registrar_key() -> Pubkey {
+pub fn get_registrar_key() -> std::result::Result<Pubkey, BatchMintError> {
+    let realm_key = Pubkey::from_str(DAO_PUBKEY)?;
```
Comment (RequescoS) [inline src/pubkey_util.rs:14]: cant we declare constant in mpl_common_constants crate via `pubkey!()` marco?
if so, they would have Pubkey type and will have no need in such from_str casts
and if we will need string representation, we can fust call to_string() method

**Review by RequescoS — APPROVED** [review]

File: Cargo.toml [hunk 2]
```diff
@@ -24,8 +24,7 @@ solana-sdk = "~1.18.11"
 solana-program = "~1.18.11"
 solana-client = "~1.18.11"
 solana-rpc-client-api = "~1.18.11"
-
-bubblegum = { git = "https://github.com/adm-metaex/mpl-bubblegum.git", rev = "afebaf7", features = ["no-entrypoint"] }
+mpl-common-constants = { git = "https://github.com/adm-metaex/mpl-common-constants.git", branch = "main", features = ["mainnet"] }
 mpl-bubblegum = { git = "https://github.com/adm-metaex/mpl-bubblegum.git", rev = "afebaf7", features = ["serde"] }
 spl-account-compression = { git = "https://github.com/StanChe/solana-program-library.git", rev = "f343436", features = ["no-entrypoint"] }
 spl-concurrent-merkle-tree = { git = "https://github.com/StanChe/solana-program-library.git", rev = "f343436", default-features = false }
```
Comment (StanChe) [inline Cargo.toml:30]: Would be great to update those as well. spl has all the changes in 0.4 crate and bubblegum is upstreamed and available in the main repo in feature/batch-mint branch

**Review by StanChe — APPROVED** [review]

File: Cargo.toml [hunk 3]
```diff
@@ -24,8 +24,7 @@ solana-sdk = "~1.18.11"
 solana-program = "~1.18.11"
 solana-client = "~1.18.11"
 solana-rpc-client-api = "~1.18.11"
-
-bubblegum = { git = "https://github.com/adm-metaex/mpl-bubblegum.git", rev = "afebaf7", features = ["no-entrypoint"] }
+mpl-common-constants = { git = "https://github.com/adm-metaex/mpl-common-constants.git", branch = "main", features = ["mainnet"] }
```
Comment (StanChe) [inline Cargo.toml:27]: how will we switch between the devnet and mainnet?

Comment (StanChe) [reply Cargo.toml:27]: Shouldn't it be a conditional import? With some default env

**Review by StanChe — COMMENTED** [review]
the feature seems to be misleading now

File: Cargo.toml (hunk 2 above)
Comment (n00m4d) [reply Cargo.toml:35]: Created separate ticket for this. There are issues appeared during that update

File: Cargo.toml (hunk 3 above)
Comment (n00m4d) [reply Cargo.toml]: Added features, like in Bubblegum

**Pull request merged**